$NetBSD: patch-doc_Makefile,v 1.1 2013/06/10 12:24:29 ryoon Exp $

* Use install-info command, instead of ginstall-info.
* Install plain (un-gzipped) info file.

--- doc/Makefile.orig	2010-04-27 06:31:00.000000000 +0000
+++ doc/Makefile
@@ -47,10 +47,10 @@ INST_TARG := $(addprefix install_, $(DOC
 
 install: $(INST_TARG)
 
-install_info: eukleides.info.gz
+install_info: eukleides.info
 	@echo "    Installing info pages"
 	@install -d $(INFO_DIR)
-	@ginstall-info --quiet $< $(INFO_DIR)/dir
+	@install-info --quiet $< $(INFO_DIR)/dir
 	@install -m 0644 $< $(INFO_DIR)
 
 install_pdf: pdf
@@ -78,7 +78,7 @@ install_nosplit: html
 uninstall:
 	@$(RM) -r $(DOC_DIR)
 ifneq ($(findstring info,$(DOC)),)
-	@ginstall-info --quiet --remove eukleides $(INFO_DIR)/dir
+	@install-info --quiet --remove eukleides $(INFO_DIR)/dir
 	@$(RM) $(INFO_DIR)/eukleides.info.gz
 endif
 
